tests/h/accounts/schemas_test.py

```diff
@@ -126,6 +126,36 @@ def test_it_is_invalid_with_invalid_characters_in_username(self,
                                                   "numbers, periods, and "
                                                   "underscores.")
 
+    def test_it_is_invalid_with_non_truthy_privacy_accepted(self,
+                                                            pyramid_request):
+        schema = schemas.RegisterSchema().bind(request=pyramid_request)
+
+        with pytest.raises(colander.Invalid) as exc:
+            schema.deserialize({"privacy_accepted": "flarp"})
```

Comment:
If you change the field to a `colander.Boolean` as I suggested above, the only falsey cases are the strings `"false"` and `"0"`, and the field being missing entirely. This is I think not what Lyza was expecting but it works fine because a `deform.CheckboxWidget` only produces the strings `"true"` or `"false"`.